src/tcgwars/logic/impl/gen8/ChillingReign.groovy

```diff
@@ -1089,11 +1089,11 @@ public enum ChillingReign implements LogicCardInfo {
       case FROSLASS_36:
         return evolution (this, from:"Snorunt", hp:HP090, type:W, retreatCost:1) {
           weakness METAL
-          bwAbility "Snowfall", {
+          bwAbility "Frost Over", {
             text "Once during your turn when you play this card from your hand to evolve a Pokémon, you may search your discard pile for a [W] Energy and attach it to 1 of your Pokémon."
             onActivate { reason->
-              if (reason == PLAY_FROM_HAND && bg.em().retrieveObject("Snowfall") != bg.turnCount && my.discard.filterByEnergyType(W) && confirm("Use Snowfall?")) {
-                bg.em().storeObject("Snowfall", bg.turnCount)
+              if (reason == PLAY_FROM_HAND && bg.em().retrieveObject("Frost Over") != bg.turnCount && my.discard.filterByEnergyType(W) && confirm("Use Frost Over?")) {
+                bg.em().storeObject("Frost Over", bg.turnCount)
```

Comment:
This is just a comment: But not sure if using whitespaces for storeObject keys will work properly. Worst case scenario, if there's some weird error please try replacing it with _ as other cards do.